Handler/Bid.hs

```diff
@@ -92,6 +106,7 @@ bidForm userId mbid = renderSematnicUiDivs $ Bid
         bidders = do
             entities <- runDB $ selectList [] [Asc UserIdent]
             optionsPairs $ map (\item -> (userIdent $ entityVal item, entityKey item)) entities
+        priceField = check validateBidPrice intField
```

Comment:
I actually solved it completely and plan a blog post about it - https://gist.github.com/amitaibu/0159947134d44627cce2d2bea221e914

we can go over it together later